recipes/pixelator/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pixelator" %}
+{% set version = "0.13.1" %}
+{% set pypi_name = "pixelgen-pixelator" %}
```

Comment:
I think that this can be removed?

Reply:
```suggestion
```

Reply:
meh, maybe it's nice to keep it, even if just as metadata for anyone viewing the recipe! 😅